### ISSUE TITLE
A11y_Model Builder_Add Data_Input_Usability : When Column to Predict dropdown is expanded users will be able to move only one value either up/down within it.

### ISSUE THREAD

**Issue by manaswita-chougule** [dotnet/machinelearning-modelbuilder#2680]
### GitHub Tags
#A11ySev4; #A11yUsable; #A11yTCS; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023;#Win32;

### Environment
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22621.1702
Narrator

### Repro Steps

1.Launch the Visual Studio with at least one of the .NET workloads.
2.Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3.Create a new .NET project or open an existing .NET project.
4.Open context menu (Right-click) of the project in the solution explorer.
5.Tab till Add and then tab till Machine Learning Model and press enter.
6.Model Builder dialog will open. Complete Scenario and environment setup steps.
7.Add Data screen will appear. Tab till Input section.
8.Tab till File Radio button and press enter to select it.
9.Tab till browse button and press enter. Upload a file for training.
10.Tab till "Column to Predict" combo-box. and Press Alt+down arrow to expand it.
11.Use up/down arrow keys to navigate within the expanded list.
12.Observe that when "Column to Predict dropdown" is expanded with arrow keys will able to move only one value either up/down within it.

### Actual Result
Unable to change the values more than once when Column to Predict dropdown is expanded. When Column to Predict dropdown is expanded users will able to move only one value either up/down within it using arrow keys .

### Expected Result
Until the dropdown is expanded user should be able to move endlessly within it to select any value via arrow keys.

### User Impact
Keyboard users will get hinderance while navigating through the dropdown list and need to repeat expand/collapse action every time to move within it.

### Attachment
![When Column to Predict dropdown is expanded users will be able to move only one value either up down within it](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/403ef417-1764-4fa1-b7c2-dc5fc368a508)

**Comment by zewditu** [comment]
This is because the combo box is editable true.

**Comment by manaswita-chougule** [comment]
@zewditu From above comment, are you suggesting that as the **combo box is editable true** it's by-design that it allow user to move only one value either up/down within it using arrow keys??

**Comment by zewditu** [comment]
No, it seems that it is WPF bug, when I make editable set to false the user is able to navigate more than one item.

**Comment by manaswita-chougule** [comment]
#Regressed:08-07-23; #PendingInput;

@zewditu As per your comment above, is this issue external to WPF team.? Please confirm us, so will route the bug to WPF team correctly. Thanks!
